src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java

```diff
@@ -114,4 +114,40 @@
     public static final AttributeKey<String>        ELASTIC_TRANSACTION_TYPE = AttributeKey.stringKey("type");
 
 
+    public static final AttributeKey<String>        CI_CLOUD_LABEL = AttributeKey.stringKey("ci.cloud.label");
+    public static final AttributeKey<String>        CI_CLOUD_NAME = AttributeKey.stringKey("ci.cloud.name");
+    /**
+     * @see <a href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/cloud.md">cloud semantic conventions</a>
```

Comment:
Would it make sense to create a dedicated class to hold these cloud attributes like `OTelCloudSemanticAttributes` ? It was misleading for me to see these attributes on the `JenkinsOtelSemanticAttributes`. I was wondering if we missed the Otel Cloud attributes.